lib/accounts_session/encoders/jwt_token.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module AccountsSession
+  module Encoders
+    module JwtToken
+      def self.encode(session)
+        JWT.encode(
+          {
+            session_id: session.session_id,
+            account: {
+              uuid: session.uuid,
+              first_name: session.first_name,
+              last_name: session.last_name,
+              email: session.email,
+              avatar_url: session.avatar_url,
+            }
```

Comment:
Maybe the session should handle formatting the data? Might be better to call `JWT.encode(session.to_h)`, if possible.